mtga_follower.py

```diff
@@ -415,6 +416,19 @@ def __extract_payload(self, blob):
 
         return blob
 
+    def __update_screen_name(self, screen_name):
+        if self.user_screen_name:
```

Comment:
I would check to see if the screen name is different from the currently set one, as people can log out and log back in

Reply:
Done